packages/cli/lib/output.ts

```diff
@@ -34,7 +34,7 @@ export interface CLISuccessMessageConfig {
 /**
  * Automatically disable styling applied by chalk if CI=true
  */
-if (process.env.CI === 'true') {
+if (process.env.CI === 'true' && !process.env.FORCE_COLOR) {
```

Comment:
This won't when someone has `FORCE_COLOR nx build` because `process.env.FORCE_COLOR` will equal `''` which is falsy.

Can you change the logic to handle this? Perhaps..

```suggestion
if (process.env.CI === 'true' && (typeof process.env.FORCE_COLOR === 'string' && process.env.FORCE_COLOR !== 'false')) {
```